src/lib/mongoBackend/connectionOperations.cpp

```diff
@@ -575,7 +578,10 @@ bool collectionCreateIndex
 
   try
   {
-    connection->createIndex(col.c_str(), indexes);
+    if (isTTL)
```

Comment:
Style: better use:

```
if (...)
{
   ...
}
```

Reply:
In addition, maybe you could elaborate (in a code comment) why ttl time (different from other indexes) needs the IndexSpec() stuff.

Reply:
Fixed in 5a4703d